Composer/packages/extensions/visual-designer/src/components/menus/IconMenu.tsx

```diff
@@ -78,6 +80,9 @@ export const IconMenu: React.FC<IconMenuProps> = ({
       },
     };
 
+    const onMenuClick = () => {
+      handleMenuShow && handleMenuShow(true);
```

Comment:
This looks like it never gets set back to false. Seems like a bug to me, but I haven't functionally tested this.

Reply:
Thanks for point out. I forgot to add callback to set menuSelected to false, fixed it.